app/Http/Controllers/Organization/FilmsController.php

```diff
@@ -53,9 +53,9 @@ public function location()
     {
         return $this->reply([
             [
-                'server'    => 'dbp-video.s3.amazonaws.com',
-                'root_path' => '',
-                'protocol'  => 'http',
+                'server'    => 'content.cdn.dbp-vid.dbp4.org',
```

Comment:
we have a mechanism for externalizing this info. suggest we use it instead of hardcoding the CDN server